### ISSUE TITLE
It's printing out options

### ISSUE THREAD

**Issue by virkt25** [imjuni/create-ts-index#8]
Version 1.5.0 is outputting all the options even when `verbose` mode isn't enabled. 

Reproduce: Run `cti create ./src` in a package.

Output: 

```
Option:  { addNewline: true,
  excludes:
  [ '@types', 'typings', '__test__', '__tests__', 'node_modules' ],
  fileExcludePatterns: [],
  fileFirst: false,
  globOptions: { cwd: './src' },
  includeCWD: true,
  quote: '\'',
  targetExts: [ 'ts', 'tsx' ],
  useSemicolon: true,
  useTimestamp: false,
  verbose: false }
```

---

I believe the offending line is: https://github.com/imjuni/create-ts-index/blob/master/src/cti.ts#L140

**Comment by imjuni** [comment]
@virkt25 Hi,

I think that is obviously bug. I will fix this weekend. 

Thank you.

**Comment by virkt25** [comment]
@imjuni I created PR #10. Hope it helps. Figured it would be nice to have a way to show the options for debugging so I opted for the `debug` module.

**Comment by imjuni** [comment]
@virkt25 Thank you for your pull-request. I think visionmedia/debug tool very useful. But I don't want verbose mode worked by operation environment variable. I think verbose option worked by option that is more useful. Nevertheless I think visionmedia/debug tool very useful for create-ts-index. So I consider next update added debug package.

And I fix this issue 1.5.1 version. Thank you for your waiting.

**Comment by virkt25** [comment]
Thanks for the fix @imjuni! Appreciate you maintaining this module.

I'm still seeing the `Options` print out in version 1.5.1. I get the following output:

```
Option:  { addNewline: true,
  excludes:
   [ '@types', 'typings', '__test__', '__tests__', 'node_modules' ],
  fileExcludePatterns: [],
  fileFirst: false,
  globOptions: { cwd: './src' },
  includeCWD: true,
  quote: '\'',
  targetExts: [ 'ts', 'tsx' ],
  useSemicolon: true,
  useTimestamp: false,
  verbose: false }
create successed
```

Is it possible to just print the success message if verbose mode is not on. 

Also I'd suggest changing `create successed` to `create succeeded` for grammar.

**Comment by imjuni** [comment]
@virkt25 

Thank you for your proposal.

I move option message to verbose option, and fix grammar in vere 1.5.2.

**Comment by virkt25** [comment]
Fix verified; thank you for your help @imjuni, awesome package!
